src/convenience.jl

```diff
@@ -1,75 +1,85 @@
-getinputs{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findinputs(coms))
-
-isfree{T<:ImagineSignal}(com::T)  = isfree(daq_channel(com), rig_name(com))
-isfixed{T<:ImagineSignal}(com::T)  = !isfree(com)
-findfree{T<:ImagineSignal}(coms::AbstractVector{T}) = find(x->isfree(x), coms)
-getfree{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findfree(coms))
-findfixed{T<:ImagineSignal}(coms::AbstractVector{T}) = find(x->!isfree(x), coms)
-getfixed{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findfixed(coms))
-
-ispos{T<:ImagineSignal}(com::T)  = ispos(daq_channel(com), rig_name(com))
-findpositioners{T<:ImagineSignal}(coms::AbstractVector{T}) = find(x->ispos(x), coms)
-getpositioners{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findpositioners(coms))
-
-isposmonitor{T<:ImagineSignal}(com::T)  = isposmonitor(daq_channel(com), rig_name(com))
-findpositionermonitors{T<:ImagineSignal}(coms::AbstractVector{T}) = find(x->isposmonitor(x), coms)
-getpositionermonitors{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findpositionermonitors(coms))
-
-iscam{T<:ImagineSignal}(com::T)  = iscam(daq_channel(com), rig_name(com))
-findcameras{T<:ImagineSignal}(coms::AbstractVector{T}) = find(x->iscam(x), coms)
-getcameras{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findcameras(coms))
-
-iscammonitor{T<:ImagineSignal}(com::T)  = iscammonitor(daq_channel(com), rig_name(com))
-findcameramonitors{T<:ImagineSignal}(coms::AbstractVector{T}) = find(x->iscammonitor(x), coms)
-getcameramonitors{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findcameramonitors(coms))
-
-islas{T<:ImagineSignal}(com::T)  = islas(daq_channel(com), rig_name(com))
-findlasers{T<:ImagineSignal}(coms::AbstractVector{T}) = find(x->islas(x), coms)
-getlasers{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findlasers(coms))
-
-isstim{T<:ImagineSignal}(com::T)  = isstim(daq_channel(com), rig_name(com))
-findstimuli{T<:ImagineSignal}(coms::AbstractVector{T}) = find(x->isstim(x), coms)
-getstimuli{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findstimuli(coms))
-
-isgalvo{T<:ImagineSignal}(com::T)  = iscam(daq_channel(com), rig_name(com))
-findgalvos{T<:ImagineSignal}(coms::AbstractVector{T}) = find(x->iscam(x), coms)
-getgalvos{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findcameras(coms))
-
-isgalvomonitor{T<:ImagineSignal}(com::T)  = isgalvomonitor(daq_channel(com), rig_name(com))
-findgalvomonitors{T<:ImagineSignal}(coms::AbstractVector{T}) = find(x->isgalvomonitor(x), coms)
-getgalvomonitors{T<:ImagineSignal}(coms::AbstractVector{T}) = view(coms, findgalvomonitors(coms))
-
-hasmonitor{T<:ImagineSignal}(com::T) = iscam(com) || ispos(com) || isgalvo(com)
-hasactuator{T<:ImagineSignal}(com::T) = iscammonitor(com) || isposmonitor(com) || isgalvomonitor(com)
-
-function monitor_name{T<:ImagineSignal}(com::T)
+finddaqchan(coms::AbstractVector{T}, nm::AbstractString) where{T<:ImagineSignal} =
```

Comment:
I usually see a space after `where` but clearly it must not be essential. It's pretty readable this way.